warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -1287,7 +1287,7 @@ protected void setupRowBasedHdfsBackedSet(String row) throws IOException {
                 this.createdRowDir = false;
             }
 
-            this.set = new HdfsBackedSortedSet<>(null, hdfsBackedSetBufferSize, ivaratorCacheDirs, row, maxOpenFiles, numRetries, persistOptions,
+            this.set = new HdfsBackedSortedSet<>(null, null, hdfsBackedSetBufferSize, ivaratorCacheDirs, row, maxOpenFiles, numRetries, persistOptions,
```

Comment:
Agreed.  Created a builder pattern.

Reply:
I have some feedback for this change in a separate branch: https://github.com/NationalSecurityAgency/datawave/pull/2065

I cleaned up the builder usage so that you don't need to override all of the base builder's methods just to call the super implementation...  That kind of defeats the purpose of having a base and extended version of the builder, does it not?

I also updated the formatting around all of the builder calls to make it easier to read.

Reply:
merged